app/models/taxon.rb

```diff
@@ -93,4 +93,8 @@ def visible_to_departmental_editors=(val)
   def visible_to_departmental_editors
     @visible_to_departmental_editors || false
   end
+
+  def root_taxon?
+    parent.nil?
```

Comment:
I think this is a bit risky, as _real_ root taxons are probably referred to by the homepage...

Reply:
Looking at some examples, I don't think they are. But do you have any suggestions to make this less risky?

Reply:
> Looking at some examples, I don't think they are. But do you have any suggestions to make this less risky?

Well, what it really tests is does this taxon have a parent (e.g. `has_parent?`). Checking if it is a root taxon requires checking the links for the homepage (which is what the GovukBranches module/class does if I remember correctly).

Reply:
Ah right, so I think I'll remove that method, and just replace it by a `@taxon.parent.nil?` check. 
The code for displaying the checkbox for this attribute also does this -> https://github.com/alphagov/content-tagger/blob/master/app/services/taxonomy/edit_page.rb#L27-L29